docs/configdoc.md

```diff
@@ -0,0 +1,117 @@
+# L3AFD Config Options Documentation
+
+See [l3afd.cfg](https://github.com/l3af-project/l3afd/blob/main/config/l3afd.cfg) for a full example configuration.
+
+
+```
+[DEFAULT]
+
+[l3afd]
+pid-file: ./l3afd.pid
+datacenter: dummy
+bpf-dir: /dev/shm
+bpf-log-dir:
+kernel-major-version: 4
+kernel-minor-version: 15
+shutdown-timeout: 1s
+http-client-timeout: 10s
+max-nf-restart-count: 3
+bpf-chaining-enabled: true
+swagger-api-enabled: false
+# PROD | DEV
+environment: PROD
+....
+```
+
+### Below is the detailed documentation for each field
+
+
+## [l3afd]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|pid-file| `"l3afd.pid"`  |It contains process id of eBPF Programs |
+|datacenter| `"dummy"` | Name of Datacenter|
+|bpf-dir| `"/dev/shm"` | Absolute Path of eBPF Programs (where they are extracted) |
+|bpf-log-dir|`""`      | Absolute Path,where we store logs eBPF Programs|
+|kernel-major-version|`"4"`|Major version of the kernel|
+|kernel-minor-version|`"15"`|Miner version of the kernel (Ex 4.15)|
+|shutdown-timeout|`"1s"`|It is the amount of time allowed to stop all network functions|
+|http-client-timeout|`"10s"`|It is the amount of time allowed to get response headers|
+|max-nf-restart-count|`"3"`|Maximum number of attempts to monitor eBPF Program|
+|bpf-chaining-enabled|`"true"`|Boolean to set bpf-chaining |
```

Comment:
Explain (or reference a document that does).  What does it mean when this is true and what does it mean when this is false?  I.e., what will l3afd do?